README.md

```diff
@@ -44,6 +133,32 @@ To test the application, run the following command:
 npm run test
 ```
 
+
+## Contributing
```

Comment:
If we keep this I think it should be the last think before the Licence.

Reply:
done

Reply:
I still see them 
![image](https://github.com/comixcraft/Editor/assets/47862956/cd4bcf1d-4d7f-467e-a0b2-422f1ab62365)

Reply:
should be good!